iot/end_to_end_example/device/app/src/main/kotlin/com/example/androidthings/coffeecontrol/BoardDefaults.kt

```diff
@@ -0,0 +1,89 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+ * Copyright 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.androidthings.coffeecontrol
+
+import android.os.Build
+
+object BoardDefaults {
+    private val DEVICE_RPI3 = "rpi3"
+    private val DEVICE_IMX6UL_PICO = "imx6ul_pico"
+    private val DEVICE_IMX7D_PICO = "imx7d_pico"
+
+    val buttonGpioPin: String
```

Comment:
You can roll these up to return the when expression to make things a bit cleaner. See the Android Things samples as an example: https://github.com/androidthings/sample-button/blob/master/kotlin/app/src/main/kotlin/com/example/androidthings/button/BoardDefaults.kt